Parking_Zone.Test/Controllers/ParkingSlotControllerTest.cs

```diff
@@ -59,8 +59,9 @@ public void GivenParkingZoneId_WhenIndexIsCalled_ThenReturnsParkingSlotsVM()
         var expectedVMs = new List<ListOfSlotsVM>();
         expectedVMs.AddRange(ListOfSlotsVM.MapToVM(_parkingSlot));
 
-        _parkingSlotServiceMoq.Setup(x => x.GetSlotsByZoneId(_id)).Returns(_parkingSlot);
-        _parkingZoneServiceMoq.Setup(x => x.GetById(_id)).Returns(_parkingZone);
```

Comment:
_parkingZoneServiceMoq.Setup(x => x.GetById(_id)).Returns(_parkingZone);
shunga ham pasda verify yozib qoying
        _parkingSlotServiceMoq.Verify(_parkingSlot => _parkingSlot.GetById(_id), Times.Once);
shunaqa

Reply:
I have not used ParkingZoneService in ParkingSlot  so I deleted it from there